procedures.md

```diff
@@ -250,6 +250,10 @@ and by hitting enter twice and leaving a blank line the subsequent text will bec
 commit message. The `git format-patch` utility will automatically put the title and message
 of a commit into the respective title and body of the corresponding generated email patch file.
 
+The commit title should include the assignment prefix followed by a colon and a space,
+then a short summary of the changes in this commit in the imperative tense.
+e.g. `E0: Add username/E0.txt`.
+
 Your commits should have a title that is a short summary of the changes in this commit and you
 should include any further details in the commit message.
```

Comment:
This sentence is duplicative of the one above. You should merge the two of them (i.e. add the part about the prefix and imperative tense to this sentence)

Reply:
I just corrected this.